plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -107,9 +107,7 @@ class SentryPlugin : Plugin<Project> {
                             sentryProperties?.let { file -> project.file(file) }
                         )
                         it.uuidDirectory.set(uuidOutputDirectory)
-                        mappingFile?.let { mapFile ->
-                            it.mappingsFile.set(mapFile)
-                        }
+                        it.mappingsFiles = mappingFileProvider
```

Comment:
would be possible to just assign `variant.mappingFileProvider` directly?

Reply:
Indeed 👍